### PR TITLE
feat(config): add Zooz ZEN73/ZEN74; minor fix to importConfig.ts

### LINKED ISSUE #2577: Zooz ZEN73 / ZEN74 support
**Which device is missing?**

Zooz ZEN73 / ZEN74.  This request is specifically for Zen74

**What are the IDs?**

Manufacturer ID: -  
Product type: 0x7000
Product ID: 0xA004  

**Did you check if we really don't have it?**

yes i checked
- [x ] It is not in the [configuration DB](https://devices.zwave-js.io/)
- [x ] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)
- [x ] It is not in the [TODO list](https://github.com/zwave-js/node-zwave-js/issues/1600)

**Do you have a link to the PDF manual?**
https://cdn.shopify.com/s/files/1/0218/7704/files/zooz-700-series-toggle-dimmer-zen74-ver1.02-manual-online.pdf

If so, it helps in creating the config file.

### PR DESCRIPTION
**Pull request by blhoward2**
Adds two new devices at the request of Zooz. Processing these revealed a minor bug to importConfig.ts, which is fixed.

fixes: #2577

### CODE REVIEW

**Comment by marcus-j-davies** [comment]
@blhoward2, do you want to link issue  https://github.com/zwave-js/node-zwave-js/issues/2577

File: packages/config/config/devices/0x027a/zen73.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+			"options": [
+				{
+					"label": "Binary switch CC report (Z-Wave); Basic CC Set (Manual)",
+					"value": 0
+				},
+				{
+					"label": "Binary switch CC report (Z-Wave / Manual)",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen73.json]:
```suggestion
			"options": [
				{
					"label": "Binary Switch CC report (Z-Wave); Basic CC Set (Manual)",
					"value": 0
				},
				{
					"label": "Binary Switch CC report (Z-Wave / Manual)",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x027a/zen73.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+				{
+					"label": "Turn off",
+					"value": 0
+				},
+				{
+					"label": "Turn on",
+					"value": 1
+				},
+				{
+					"label": "Previous State",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen73.json]:
```suggestion
					"label": "Previous state",
```

File: packages/config/config/devices/0x027a/zen73.json [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+				{
+					"label": "Physical control disabled",
+					"value": 0
+				},
+				{
+					"label": "Physical control enabled",
+					"value": 1
+				},
+				{
+					"label": "Physical and Z-Wave Control Disabled",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen73.json]:
```suggestion
					"label": "Physical and Z-Wave control disabled",
```

File: packages/config/config/devices/0x027a/zen74.json [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+				{
+					"label": "Turn off",
+					"value": 0
+				},
+				{
+					"label": "Turn on",
+					"value": 1
+				},
+				{
+					"label": "Previous State",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen74.json]:
```suggestion
					"label": "Previous state",
```

File: packages/config/config/devices/0x027a/zen74.json [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+				{
+					"label": "Physical control disabled",
+					"value": 0
+				},
+				{
+					"label": "Physical control enabled",
+					"value": 1
+				},
+				{
+					"label": "Physical and Z-Wave Control Disabled",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen74.json]:
```suggestion
					"label": "Physical and Z-Wave control disabled",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
:)

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**